src/vs/workbench/browser/actions/windowActions.ts

```diff
@@ -342,6 +360,7 @@ registry.registerWorkbenchAction(SyncActionDescriptor.from(QuickPickRecentAction
 registry.registerWorkbenchAction(SyncActionDescriptor.from(OpenRecentAction, { primary: KeyMod.CtrlCmd | KeyCode.KEY_R, mac: { primary: KeyMod.WinCtrl | KeyCode.KEY_R } }), 'File: Open Recent...', fileCategory);
 
 registry.registerWorkbenchAction(SyncActionDescriptor.from(ToggleFullScreenAction, { primary: KeyCode.F11, mac: { primary: KeyMod.CtrlCmd | KeyMod.WinCtrl | KeyCode.KEY_F } }), 'View: Toggle Full Screen', CATEGORIES.View.value);
+registry.registerWorkbenchAction(SyncActionDescriptor.from(ExitFullScreenAction, { primary: KeyCode.Escape }), 'View: Exit Full Screen', CATEGORIES.View.value);
```

Comment:
We cannot bind `Escape` key per default, this will be very frustrating for users that want to stay in fullscreen and hit the Escape key. I think it is good to register the action to appear in the F1 list, but not with such a prominent keybinding.